src/Cli/dotnet/commands/dotnet-test/TestCommandParser.cs

```diff
@@ -77,7 +77,7 @@ internal static class TestCommandParser
         public static readonly Option<IEnumerable<string>> CollectOption = new ForwardedOption<IEnumerable<string>>("--collect", LocalizableStrings.cmdCollectDescription)
         {
             ArgumentHelpName = LocalizableStrings.cmdCollectFriendlyName
-        }.ForwardAsSingle(o => $"-property:VSTestCollect=\"{string.Join(";", o)}\"")
+        }.ForwardAsSingle(o => $"-property:VSTestCollect=\"{string.Join(";", GetSemiColonEscapedArgs(o))}\"")
```

Comment:
I am investigating a bug where having commas can break the command line, could we have commas here?

Reply:
I believe you mean commas instead of semi colon here. In our workflow we cannot.
Example:
--collect "Code Coverage;arg1=something,anything;arg2=val2
We can provide multiple settings (arg1, arg2) separated by semi colons. And each setting can have multiple values (arg1=something,anything).

Reply:
My point was more to say, is it possible to have one of the collector name with a comma in its name (e.g. if that's done with F# through ```My collector, bla bla```.

Reply:
So I double checked and there is no problem because the whole property is escaped with double quotes.